clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/DefaultAmazonAccountsSynchronizer.groovy

```diff
@@ -1,62 +0,0 @@
-/*
- * Copyright 2020 Armory
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.clouddriver.aws.security
-
-import com.netflix.spinnaker.cats.module.CatsModule
-import com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsConfig
-import com.netflix.spinnaker.clouddriver.aws.security.config.CredentialsLoader
-import com.netflix.spinnaker.clouddriver.security.AccountCredentialsRepository
-import com.netflix.spinnaker.clouddriver.security.ProviderUtils
-
-import static com.amazonaws.regions.Regions.*
-
-class DefaultAmazonAccountsSynchronizer implements AmazonAccountsSynchronizer {
-
-  List<? extends NetflixAmazonCredentials> synchronize(
-    CredentialsLoader<? extends NetflixAmazonCredentials> credentialsLoader,
-    CredentialsConfig credentialsConfig,
-    AccountCredentialsRepository accountCredentialsRepository,
-    DefaultAccountConfigurationProperties defaultAccountConfigurationProperties,
-    CatsModule catsModule) {
-    if (!credentialsConfig.accounts && !credentialsConfig.defaultAssumeRole) {
-      def defaultEnvironment = defaultAccountConfigurationProperties.environment ?: defaultAccountConfigurationProperties.env
-      def defaultAccountType = defaultAccountConfigurationProperties.accountType ?: defaultAccountConfigurationProperties.env
-      credentialsConfig.accounts = [new CredentialsConfig.Account(name: defaultAccountConfigurationProperties.env, environment: defaultEnvironment, accountType: defaultAccountType)]
-      if (!credentialsConfig.defaultRegions) {
-        credentialsConfig.defaultRegions = [US_EAST_1, US_WEST_1, US_WEST_2, EU_WEST_1].collect {
-          new CredentialsConfig.Region(name: it.name)
-        }
-      }
-    }
-
-    List<? extends NetflixAmazonCredentials> accounts = credentialsLoader.load(credentialsConfig)
-
-    def (ArrayList<NetflixAmazonCredentials> accountsToAdd, List<String> namesOfDeletedAccounts) =
-    ProviderUtils.calculateAccountDeltas(accountCredentialsRepository, NetflixAmazonCredentials, accounts)
```

Comment:
As an operator, it's helpful to see in the logs what accounts/credentials are being successfully discovered by clouddriver in the event I have to debug something. `ProviderUtils.calculateAccountDeltas` provides some nice logging here:
```
// example artifact acct

2020-11-06 18:59:00.890  INFO 23693 --- [           main] c.n.s.c.security.ProviderUtils           : Adding accounts [my-ca-central-1-devel-regis
try] of type DockerRegistryNamedAccountCredentials...
2020-11-06 18:59:02.345  INFO 23693 --- [           main] c.n.s.c.a.c.BaseHttpArtifactCredentials  : Loaded credentials for artifact account my-g
ithub-acct
```

...which we're losing by eliminating it. I understand the frequency of account adds/removals would become more frequent with a dynamic accounts plugin enabled, but also more important to document in the event something is messed up. 

Can we add similar logging to `EcsCredentialsLifeCycleHandler` re: account additions / removals?

Reply:
Added logging in `EcsCredentialsLifeCycleHandler`: 
```
2020-11-06 15:38:14.871  INFO 84556 --- [           main] n.s.c.e.s.EcsCredentialsLifeCycleHandler : ECS account, mccloman-1-ecs, was added. Scheduling caching agents
```